domains.js

```diff
@@ -0,0 +1,72 @@
+const google = require('googleapis');
+const sheets = google.sheets('v4');
+
+function getDomains(username) {
+    const credentials = credentialConfiguration();
+    const request = {
+        spreadsheetId: credentials.TASK_SHEET_ID,
+        range: 'A:ZZZ', // extreme range to get whole spreadsheet
```

Comment:
`spreadsheets.get()` with grid data returns an object nested with obscene amounts of metadata that I don't want to slog through. `values.get()` is just an array of values, much easier to work with. This was the recommended method from my own research. The api doesn't return empty cells, so there's no worry about extraneous information.